<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatWidget</class>
 <widget class="QWidget" name="chatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chatWidget</string>
  </property>
  <widget class="QTextBrowser" name="msgBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>550</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="msgTxtEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>550</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="usrTblWidget">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>260</width>
     <height>400</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>129</number>
   </attribute>
   <column>
    <property name="text">
     <string>用户</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>IP地址</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>420</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontCbx">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="sizeCbx">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
  </widget>
  <widget class="QToolButton" name="clearTBtn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="saveTBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="colorTBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="sendTBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="underlineTBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="italicTBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="boldTBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="usrNumLbl">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>420</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>在线用户：0人</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
